packages/gestalt/src/IconButton.js

```diff
@@ -184,6 +186,7 @@ const IconButtonWithForwardRef: React$AbstractComponent<
         onMouseLeave={handleOnMouseLeave}
         ref={innerRef}
         rel={rel}
+        tabIndex={tabIndex}
```

Comment:
@AlbertCarreras we should also add `tabIndex` to the `<button />` element below

Reply:
done